src/engine/compiler/MacroAssembler.v3

```diff
@@ -311,6 +323,27 @@ class MacroAssembler(valuerep: Tagging, regConfig: RegConfig) {
 
 	def emit_debugger_breakpoint();
 
+	// Gets the number of parameters to a continuation.
```

Comment:
I think I'd prefer to see this one inlined at the call sites, otherwise there is a lot of jumping backing and forth to the Masm.